ui_testing/pages/base_pages.py

```diff
@@ -112,12 +113,13 @@ def select_random_table_row(self, element='general:table'):
         self.info("select random row")
         rows = self.base_selenium.get_table_rows(element=element)
         for _ in range(5):
-            row = rows[randint(0, len(rows) - 1)]
+            row_index = randint(0, len(rows) - 1)
+            row = rows[row_index]
             row_text = row.text
             if not row_text:
                 continue
             self.click_check_box(source=row)
-            return self.base_selenium.get_row_cells_dict_related_to_header(row)
+            return self.base_selenium.get_row_cells_dict_related_to_header(row), row_index
```

Comment:
changing the return method will affect other test cases that already use this method. Then u need to update all of them

Reply:
I think adding a second return value won't affect any other cases, because the first value returned is always the same.

Reply:
![image](https://user-images.githubusercontent.com/17981089/71575808-98a8be00-2aee-11ea-8942-97cb9c035fc5.png)

Reply:
Done